### PR TITLE
Feature/client login

### LINKED ISSUE #10: As a client, I want to be able to create a client account (epic-account).

### PR DESCRIPTION
**Pull request by dmercanti**

### CODE REVIEW

File: sadcowpng/sharedspaces/templates/sharedspaces/client_sign_up.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    <p>Here is where you can create an account to
+        start finding your perfect new hangout spot!</p>
+    <h2>Sign Up Now!</h2>
+    <form method="post">
+        {% csrf_token %}
+        {{form}}
+        <button type="submit">Sign Up</button>
+    </form>
+    </body>
+</html>
```
Comment (sharletclaros) [inline sadcowpng/sharedspaces/templates/sharedspaces/client_sign_up.html:20]: Newline at end of file needed. Can be changed anytime.

**Review by sharletclaros — APPROVED** [review]
Minimal syntax changes needed. Everything else LGTM.

**Review by sharletclaros — DISMISSED** [review]

File: sadcowpng/sharedspaces/templates/sharedspaces/signup.html [hunk 2]
```diff
@@ -1,10 +1,39 @@
...
 <body>
     <p>Sign up to create an account for shared spaces.</p>
+
+    <div class="container">
+        <h1> Sign Up </h1>
+        <div class="float-c">
+            <h5>Client Account</h5>
+            <p>"Description"</p>
+
+            <a href="#">
```
Comment (btruong9) [inline sadcowpng/sharedspaces/templates/sharedspaces/signup.html]: The client sign up url should be add in place of the '#'.

File: sadcowpng/sharedspaces/views.py [hunk 3]
```diff
@@ -16,6 +23,20 @@ def login(request):
     return render(request, 'sharedspaces/login.html')
 
 
+# Proprietor sign up view
```
Comment (btruong9) [inline sadcowpng/sharedspaces/views.py]: Comment should be client instead of proprietor

**Review by jimmy-coleman — APPROVED** [review]
lgtm

**Review by bishalr1 — APPROVED** [review]
This looks very good to me.

**Review by btruong9 — APPROVED** [review]
LGTM!

**Pull request merged**